testing.rst

```diff
@@ -505,7 +505,7 @@ afterwards with the ``followRedirect()`` method::
     $crawler = $client->followRedirect();
 
 If you want the client to automatically follow all redirects, you can
```

Comment:
@ShinDarth thanks for this improvement. I like it ... but I still think that the original phrase is not very good. What if we reword it as follows:

```
You can make the client to automatically follow all redirects by
calling the ``followRedirects()`` method before making the first request::
```